app/views/manage_assessments/outcome_coverages/_outcome_coverage.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2><%= outcome_coverage.subject_title %></h2>
```

Comment:
Should we now remove the corresponding delegate (`delegate :title, to: :subject, prefix: true`) in the outcome_coverage model as it is no longer being used?